src/chainstate/burn/db/sortdb.rs

```diff
@@ -470,22 +471,19 @@ const BURNDB_SETUP: &'static [&'static str] = &[
 
         PRIMARY KEY(sortition_id)
     );"#,
-    r#"
-    CREATE UNIQUE INDEX snapshots_block_hashes ON snapshots(block_height,index_root,winning_stacks_block_hash);
-    CREATE UNIQUE INDEX snapshots_block_stacks_hashes ON snapshots(num_sortitions,index_root,winning_stacks_block_hash);
-    CREATE UNIQUE INDEX snapshots_block_heights ON snapshots(burn_header_hash,block_height);
-    CREATE UNIQUE INDEX snapshots_burn_hashes ON snapshots(block_height,burn_header_hash)
-    CREATE INDEX snapshots_block_winning_hash ON snapshots(winning_stacks_block_hash);
-    CREATE INDEX block_arrivals ON snapshots(arrival_index,burn_header_hash);
-    CREATE INDEX arrival_indexes ON snapshots(arrival_index);
-    "#,
+    "CREATE UNIQUE INDEX snapshots_block_hashes ON snapshots(block_height,index_root,winning_stacks_block_hash);",
+    "CREATE UNIQUE INDEX snapshots_block_stacks_hashes ON snapshots(num_sortitions,index_root,winning_stacks_block_hash);",
```

Comment:
I'm not sure that these tuples are unique in the presence of PoX -- if there were two PoX forks that both picked the same "winner" at the same height (I believe that is possible), then this constraint would be violated. I think that these indexes should drop the "unique" constraint.

Then, it seems like:

```
    "CREATE INDEX snapshots_block_heights ON snapshots(burn_header_hash,block_height);",
    "CREATE INDEX snapshots_burn_hashes ON snapshots(block_height,burn_header_hash);",
```

are both indexing more or less the same thing, so I'm pretty sure that one of them can be removed without affecting performance (and reducing the number of indexes managed by the db).

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2420/commits/85822097eb7f13969a83f4d096df208b50896db7

Reply:
Please restore this index:

```
 CREATE INDEX snapshots_block_winning_hash ON snapshots(winning_stacks_block_hash);
 ```